Admin/Admin.php

```diff
@@ -292,7 +292,7 @@
      * then the $parentReflectionProperty must be the ReflectionProperty of
      * the order (OrderElement::$order).
      *
-     * @var \ReflectionProperty
+     * @var mixed
```

Comment:
Looked at the history, and it seems that the comment is outdated since b0df44f9894b08ea1ac526231990547919da375f